Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesVfr.vfr

```diff
@@ -80,6 +80,12 @@ formset
              flags  = INTERACTIVE,
     endcheckbox;
 
+    checkbox varid  = FeaturesData.DmaProtection,
+             prompt = STRING_TOKEN(STR_DMA_BOOT_ENABLE_PROMPT),
+             help   = STRING_TOKEN(STR_DMA_BOOT_ENABLE_HELP),
+             flags  = INTERACTIVE,
+    endcheckbox;
+

```

Comment:
What about IOMMU handoff at ExitBootServices?

Reply:
Added in the last commit.